Sources/SwiftBasicFormat/BasicFormat.swift

```diff
@@ -12,6 +12,13 @@
 
 import SwiftSyntax
 
+/// A rewriter that performs a "basic" format of the passed tree. The base
+/// implementation is primarily aimed at adding whitespace where required such
+/// that re-parsing the tree's description results in the same tree. But it
+/// also makes an attempt at adding in formatting, eg. splitting lines where
+/// obvious and some basic indentation at nesting levels.
+///
+/// Any subclasses *must* return the same node type as given.
```

Comment:
I know that I have been very bad at this as well but docc likes to have a short explanation on the first paragraph with a detailed explanation on the following paragraphs. So, in docc this will render more nicely if written as

```suggestion
/// A rewriter that performs a "basic" format of the passed tree. 

/// The base implementation is primarily aimed at adding whitespace where required such
/// that re-parsing the tree's description results in the same tree. But it
/// also makes an attempt at adding in formatting, eg. splitting lines where
/// obvious and some basic indentation at nesting levels.
///
/// Any subclasses *must* return the same node type as given.
```

Reply:
Ah, sure thing.